backend/OR-Tools/OR-Tools/ORSolver.cs

```diff
@@ -1,30 +1,32 @@
 ﻿using System;
 using Google.OrTools.ConstraintSolver;
 
+using Domains.Models.Output;
+
 namespace OR_Tools
 {
-    class ORSolver
+    public class ORSolver
     {
-        static readonly long penalty = 1000;
-        const int MaximumTimeForTheCar = 120;
-
-        RoutingIndexManager manager;
-        RoutingModel routing;
-        Assignment solution;
-        Data data;
-        
+        public static readonly long Penalty = 1000;
+        public const int MaximumTimeForTheCar = 120;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this constant to a 'static' read-only property.](https://app.codacy.com/app/iamprovidence/CVRPWT/pullRequest?prid=4619036)